integration/chat-plugin/BUILD

```diff
@@ -8,6 +8,7 @@ ts_web_library(
     srcs = glob(["*.spec.ts"]),
     deps = [
         "//frontend/chat-plugin/handles:chat-plugin-handles",
+        "//frontend/ui/handles",
```

Comment:
we don't need this dependency

Reply:
The build failed when I removed this dependency

Reply:
right sorry!